tests/resources/layered_decorators.py

```diff
@@ -0,0 +1,32 @@
+from hamilton.function_modifiers import config, does, parametrized
+
+def _sum(**kwargs: int) -> int:
+    return sum(kwargs.values())
+
+
+@does(_sum)
+@parametrized(parameter='a', assigned_output={('e', 'First value'): 10, ('f', 'First value'): 20})
+@config.when(foo='bar')
+def c__foobar(a: int, b: int) -> int:
+    """Demonstrates utilizing a bunch of decorators.
+    In all, this outputs two total nodes.
+    - config.when makes it only apply when foo=bar
+    - @does makes it do the sum pattern
+    - @parametrized curries the function then turns it into two
```

Comment:
say what the names of the two nodes will be?
```suggestion
    - @parametrized curries the function then turns it into two; one named `e` and the other `f`.
```